package.json

```diff
@@ -123,4 +124,4 @@
     "web3-utils": "^1.2.8",
     "yargs": "^15.3.1"
   }
-}
+}
```

Comment:
nit: missing newline at EOF

Reply:
You know I can't change this.

Reply:
I have found a way to disable JSON formatting on save.